pipeline/run_benchmark.sh

```diff
@@ -97,6 +98,11 @@ function do_simple_cmd
       if [ "$LIBP2P" == "true" ]; then
          benchmarkArgs+=" -is_genesis"
       fi
+      case "${commit_delay+set}" in
```

Comment:
set here means?

Reply:
From the “Parameter Expansion” section of `bash(1)`:


       When  not  performing  substring  expansion, using the forms documented
       below (e.g., :-), bash tests for a parameter that  is  unset  or  null.
       Omitting  the  colon  results  in  a  test only for a parameter that is
       unset.

       ...

       ${parameter:+word}
              Use  Alternate Value.  If parameter is null or unset, nothing is
              substituted, otherwise the expansion of word is substituted.

Reply:
So, "${commit_delay+set}" will be equal to `"set"` if `commit_delay` has been set (by the `d)` case of the `case` statement above), and `""` if not.